composer/loggers/__init__.py

```diff
@@ -24,13 +25,6 @@
 
 # All needs to be defined properly for sphinx autosummary
 __all__ = [
-    'Logger',
-    'LoggerDestination',
-    'FileLogger',
-    'InMemoryLogger',
-    'ProgressBarLogger',
-    'WandBLogger',
-    'RemoteUploaderDownloader',
-    'TensorboardLogger',
-    'CometMLLogger',
+    'Logger', 'LoggerDestination', 'FileLogger', 'InMemoryLogger', 'ProgressBarLogger', 'WandBLogger',
+    'RemoteUploaderDownloader', 'TensorboardLogger', 'CometMLLogger', 'ConsoleLogger'
```

Comment:
nit: would be nice to leave trailing comma in

Reply:
done